src/app/cli/CommandLineInterface.java

```diff
@@ -63,15 +92,280 @@ public void launch(){
                     exit = true;
                 }
                 case LOGIN -> {
-                    User user = app.logIn(tokens[1], tokens[2]);
-                    System.out.println(user);
+                    this.user = app.logIn(tokens[1], tokens[2]);
+                    System.out.println(this.user);
                 }
                 case LOGOUT -> {
                     app.logOut();
                 }
                 case SIGNUP -> {
-                    User user = app.signUp(tokens[1], tokens[2], tokens[3], tokens[4], tokens[5]);
-                    System.out.println(user);
+                    this.user = app.signUp(tokens[1], tokens[2], tokens[3], tokens[4], tokens[5]);
+                    System.out.println(this.user);
+                }
+
+                case GET_COLLECTIONS -> {
+                    Collection[] collections = app.get_collections();
+                    if(collections.length == 0) {
+                        System.out.println("You do not have any collections.");
+                        continue;
+                    }
+                    for(int i = 0; i < collections.length; ++i){
```

Comment:
should use an actual java sort, probably implement `Comparable` for `Collection`

Reply:
yeah but like this is bad
ig it doesnt matter that much